class/defaults.yml

```
@@ -2,6 +2,7 @@ parameters:
   thanos:
     multi_instance: true
     namespace: syn-thanos
+    namespaceDisabled: false
```

I'd go with the following instead (I always stumble over double negatives):

```suggestion
    createNamespace: true
```